32blit-sdl/Input.hpp

```diff
@@ -1,12 +1,21 @@
 #include <map>
+#include <vector>
 class System;
 
+struct game_controller
```

Comment:
Should be CamelCase I think. (Maybe even private to `Input`)

Reply:
Thought about making it private as well. But since not all functions are within the Input class, it currently needs to be public.

Reply:
And re CamelCase, nopes, see File.hpp

Reply:
The struct in that file is CamelCase though? I know there's `packed_image`, but I think that was missed back when everything was changed to CamelCase...